INSTALLATION.md

```diff
@@ -0,0 +1,270 @@
+   in the following command.
+   - To quickly get started, you can apply our manifest to generate a
+     self-signed `Issuer` that can generate a certificate for the registry
+     server.
+   ```bash
+   $ kubectl apply -f ./manifests/connectivity-registry-certs/ca_certificate.yaml
+   ```
+   - You can also use your own CA certificate to generate certificates for the
+   registry server. Create a secret named `registry-ca` in the
+   `cross-cluster-connectivity` namespace as per these
+   [instructions](https://cert-manager.io/docs/configuration/ca/).
+
+1. Change the `REGISTRY_DOMAIN` in the
+   `./manifests/connectivity-registry-certs/registry_certificate.yaml` to a
+   domain of your choosing. This is the domain where the registry server will be
+   published within the cluster. For example, the Kind cluster setup uses
+   `shared-services-registry.xcc.test`.
+
+1. Create connectivity-registry certificate.
+   ```bash
+   $ kubectl apply -f ./manifests/connectivity-registry-certs/registry_certificate.yaml
+   ```
+
+#### Install Cross-cluster Connectivity components
+
+1. Install the CRDs.
+   ```bash
+   $ kubectl apply -f ./manifests/crds
+   ```
+
+1. Install connectivity-publisher.
+   ```bash
+   $ kubectl apply -f ./manifests/connectivity-publisher
+   ```
+
+1. Install connectivity-registry.
+   ```bash
+   $ kubectl apply -f ./manifests/connectivity-registry
+   ```
+
+### Install Cross-cluster Connectivity on the Cluster consuming Services
+
+#### Setup Steps
+
+1. Create a namespace called `cross-cluster-connectivity`. This namespace will
+   be used by all the Cross-cluster Connectivity components.
+  ```bash
+  $ kubectl create namespace cross-cluster-connectivity
+  ```
+
+#### Generate Certificates for the `connectivity-registry` components
+
+To enable certificate management for the `connectivity-registry` component the
+instructions will also install `cert-manager`, but this could be replaced by
+another certificate management solution, if desired.
+
+1. Follow instructions
+   [here](https://cert-manager.io/docs/installation/kubernetes) to install
+   cert-manager.
+
+1. Add a cert-manager `Issuer` to generate a certificate for the registry server
+   in the following command.
+   - To quickly get started, you can apply our manifest to generate a
+     self-signed `Issuer` that can generate a certificate for the registry
+     server.
+   ```bash
+   $ kubectl apply -f ./manifests/connectivity-registry-certs/ca_certificate.yaml
+   ```
+   - You can also use your own CA certificate to generate certificates for the
+   registry server. Create a secret named `registry-ca` in the
+   `cross-cluster-connectivity` namespace as per these
+   [instructions](https://cert-manager.io/docs/configuration/ca/).
+
+1. Change the `REGISTRY_DOMAIN` in the
+   `./manifests/connectivity-registry-certs/registry_certificate.yaml` to a
+   domain of your choosing. This is the domain where the registry server will be
```

Comment:
The wording here is a little awkward. Should it mention that the domain name is where the registry server can be reached?